crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -207,9 +262,9 @@ mod tests {
         let (circuit, public_inputs) = StateUpdateBuilder::<F>::build(
             &st,
             &[],
-            &LightClientState::default(),
+            &lightclient_state,
             &bit_vec,
-            &U256::from(25u32),
+            &U256::from(26u32),
```

Comment:
Maybe 25 is fine as threshold is not supposed to be exactly equal to the signed weight. (Or maybe use both 26 and a threshold less than 26).

Reply:
Added one in 9927ac8